_includes/header.html

```diff
@@ -8,6 +8,12 @@
          <img class="logo-cog" alt="Mechanical Rock Cog" src="/img/logo-cog.png">
          <span>Mechanical Rock<br />Blogs</span>
       </a>
+
+      <a href="https://mechanicalrock.io/" class="logo font-brand">
+         <img class="logo-cog" alt="Mechanical Rock Cog" src="/img/logo-cog.png">
+         <span>Mechanical Rock<br />Home</span>
+      </a>
```

Comment:
Seems strange to have 2 company logo links on the top-left. 
I prefer Kaz's suggestion where the page layout contained a back-chevron link labelled "< view more blogs", and we then update the top-left link to take users to our website.